cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -40,6 +55,8 @@ virt-install \
   --network default \
   --import
 
+wait_for_install_to_complete
+
```

Comment:
I fail to see any impact with running in a local environment where the TTY is available: the script is only run once and it has no impact on the execution time because the loop will exit right away: You will only see the condition being checked once in the log. I'm not convinced this flag will add any improvement, but if you are keen to have a cleaner output I will add it there :smile:

Reply:
no worries :smile: